modules/documents/client/directives/documents.manager.link.directive.js

```diff
@@ -208,6 +208,7 @@ angular.module('documents')
       restrict: 'A',
```

Comment:
This file contains the latest changes: I had to re-add the "target" file list in order to track the existing files and re-add them when necessary (otherwise adding a new file would "clear" the previous list completely).

Reply:
cool beans